tests/has/const.go

```diff
@@ -18,6 +18,21 @@ const (
 	// Sample devfile created redhat-appstudio-qe repository with the following content:
```

Comment:
Could we change the name of the appstudio application and component? We saw a [test failure related to this](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-appstudio_e2e-tests/90/pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e/1529405386480685056)in the previous PR - the Application CR was still in terminating state and other test was failing to create the Application CR with the same name.

Reply:
Updated to use generated names for the application and component.

I used the following:
```
		applicationName = fmt.Sprintf(RedHatAppStudioApplicationName+"-%s", util.GenerateRandomString(10))
		componentName = fmt.Sprintf(QuarkusComponentName+"-%s", util.GenerateRandomString(10))
```